db/migrations/20230821214555_add_podcasts_and_episodes.js

```diff
@@ -0,0 +1,47 @@
+exports.up = function (knex) {
+  return knex.schema
+    .createTable("podcast", function (table) {
+      table.uuid("id").primary().unique();
+      table.string("user_id").notNullable().index("idx_podcast_user_id");
+      table.foreign("user_id").references("user.id");
+      table.string("name").notNullable();
+      table.string("artwork_url");
+      table
+        .string("podcast_url")
+        .notNullable()
+        .unique()
+        .index("idx_podcast_url");
+      table.string("description");
```

Comment:
We should add a length here as the `string` type defaults to 255 chars. We use 1000 for album descriptions but could be more here if we want.

https://knexjs.org/guide/schema-builder.html#string

```suggestion
      table.string("description",  1000);
```

Reply:
608133f9705eeb676d5f7e612299165b6c31cdf9